articles/app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Post;
+use App\Models\Category;
+use App\Models\Tag;
+use File;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $post = Post::OrderBy('id','DESC')->get();
+        return view('post.index',compact('post'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $category = Category::get();
```

Comment:
@FareenaEhtesham for readability, you should use plural tense when there are multiple entries, so `$categories` instead of `$category` here. Cause in the future when someone else is reading your code, they may be confused that it's a single category.